proto/ibc/client/genesis.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+package ibc.client;
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/ibc/02-client/types";
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+
+// GenesisClientState defines an identified ClientState as protobuf Any format.
+message GenesisClientState {
+  string client_id = 1 [
+    (gogoproto.customname) = "ClientID",
+    (gogoproto.moretags) = "yaml:\"client_id\""
+  ];
+  google.protobuf.Any client_state = 2 [(gogoproto.moretags) = "yaml:\"client_state\""];
```

Comment:
is yaml needed here? I thought we only need this when it differs from the proto name?

Reply:
the default yaml is camelCase, so we need to change it to snake_case